### PR TITLE
fix: typos

### PR DESCRIPTION
**Pull request by martinconic**
Used `codespell` tool to find typos and corrected them

### CODE REVIEW

File: src/io/windows.zig [hunk 1]
```diff
@@ -88,7 +88,7 @@ pub const IO = struct {
                 timeout_ms = if (expires == os.windows.INFINITE) expires - 1 else expires;
             }
 
-            // Poll for IO iff theres IO pending and flush_timeouts() found no ready completions
```
Comment (matklad) [inline src/io/windows.zig:91]: not a typo

**Review by matklad — COMMENTED** [review]
LGTM except for an `iff`, that's a perfectly fine word with a specific meaning: if and only if!

File: src/io/windows.zig (hunk 1 above)
Comment (martinconic) [reply src/io/windows.zig:91]: reverted

**Comment by martinconic** [comment]
> LGTM except for an `iff`, that's a perfectly fine word with a specific meaning: if and only if!

reverted

**Review by matklad — DISMISSED** [review]

**Comment by matklad** [comment]
Thanks!

**Review by matklad — APPROVED** [review]

**Pull request merged**